package.json

```diff
@@ -48,11 +48,12 @@
     "webpack:prod": "lerna run webpack:prod",
     "webpack:prod:web": "lerna run webpack:prod:web",
     "webpack:prod:node": "lerna run webpack:prod:node",
-    "test:all": "tap --ts --node-arg=--max-old-space-size=4096 --timeout=3600 --branches=45 --functions=70 --lines=75 --statements=75 \"packages/cactus-*/src/test/typescript/{unit,integration,benchmark}/\"",
-    "test:unit": "tap --ts --node-arg=--max-old-space-size=4096 --timeout=600 --no-check-coverage \"packages/cactus-*/src/test/typescript/unit/\"",
+    "test:all": "tap --ts --node-arg=--max-old-space-size=4096 --timeout=3600 --branches=45 --functions=70 --lines=75 --statements=75 \"packages/cactus-*/src/test/typescript/{unit,integration,benchmark}/\" ",
+    "test:unit": "tap --ts --node-arg=--max-old-space-size=4096 --timeout=600 \"packages/cactus-*/src/test/typescript/unit/\" ",
     "test:benchmark": "tap --ts --jobs=1 --no-timeout --no-check-coverage \"packages/cactus-*/src/test/typescript/benchmark/\"",
     "test:browser": "karma start karma.conf.js",
     "test:integration": "tap --ts --node-arg=--max-old-space-size=4096 --jobs=1 --timeout=3600 --no-check-coverage \"packages/cactus-*/src/test/typescript/integration/\"",
+    "posttest:all": "tap --coverage-report=lcov && codecov",
```

Comment:
@travis-payne Right now in the readme the badge render 'unknown' as the coverage value. Is there any way to verify in advance that it actually works, e.g. once we merge it won't remain 'unknown'?

Reply:
It says unknown because there hasn't been a commit on this repo yet, once we merge it should change.

Reply:
@travis-payne Understood, thanks for confirming!